BadgeFramework/badge.py

```diff
@@ -3,7 +3,7 @@
 import logging
 import sys
 import struct
-import Queue
+import queue
```

Comment:
How about importing it like this? Does that makes more sense and make the code learner?
```suggestion
from queue import Queue
```

If so, can you do the same for the other files?